pkg/status/status.go

```diff
@@ -146,6 +167,8 @@ func GetStatuses(status *common.Status) map[string]*string {
 	if placementStatus != "" {
 		statuses["PlacementStatus"] = &placementStatus
 	}
+	ustate := status.GetUploadStatus().String()
```

Comment:
Just to keep it consistent with the changes being made in the hydration PR, could we null check the result of status.GetUploadStatus like what's done here: https://github.com/microsoft/moc/pull/237/files#diff-1deea3647518183ee4915dd3c10ccfc7bb60e1ed860673c99eefaf5cfeefaa6b

Reply:
Done!